test/cypress/integration/group-payments.spec.js

```diff
@@ -220,7 +220,7 @@ describe('Payments', () => {
   })
 
   it('user2 pledges $100 and sees their contributions.', () => {
-    cy.giSwitchUser(`user2-${userId}`, { bypassUI: true })
+    cy.giSwitchUser(`user2-${userId}`)
```

Comment:
Note: By default giSwitchUser always uses `bypassUI`, so there's no need to pass the argument.